hive-metastore/src/main/java/org/apache/iceberg/hive/CachedClientPool.java

```diff
@@ -97,10 +99,15 @@ HiveClientPool clientPool() {
 
   private synchronized void init() {
     if (clientPoolCache == null) {
+      // Since Caffeine does not ensure that removalListener will be involved after expiration
+      // We use a scheduler with 2 threads to clean up expired clients.
```

Comment:
Why this is size 2? Why not 1?

Reply:
I agree that this should be rather 1 instead of 2

Reply:
OK

Reply:
Could you please update the comment?

Reply:
Thanks for your correction. I have updated the comment.